packages/plugin-ext/src/main/browser/notebooks/renderers/cell-output-webview.tsx

```diff
@@ -38,15 +38,82 @@ import { NotebookModel } from '@theia/notebook/lib/browser/view-model/notebook-m
 
 const CellModel = Symbol('CellModel');
 const Notebook = Symbol('NotebookModel');
+export const AdditionalOutputCss = Symbol('AdditionalOutputCss');
```

Comment:
Minor: This is a fairly non-descriptive name. Maybe we should use something more wordy?
```suggestion
export const AdditionalNotebookCellOutputCss = Symbol('AdditionalNotebookCellOutputCss');
```